chart/openfaas/templates/psp.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.psp }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: {{ .Release.Name }}-psp
+    labels:
+        app: {{ template "openfaas.name" . }}
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    annotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+    privileged: false
+    hostIPC: false
+    hostNetwork: false
+    hostPID: false
+    readOnlyRootFilesystem: false
+    allowPrivilegeEscalation: false
+    allowedCapabilities:
+        - '*'
```

Comment:
`    allowedCapabilities:`

Is this the same as privileged / root? If I have all Linux caps then I can do anything.

It looks similar to a Kubernetes RBAC of "*".

I haven't been close to PSP so you'll have to let me know if I'm wrong here.

Reply:
Technically OpenFaaS never adds any Linux caps to run, so I wonder if this can be blank? Perhaps Istio needs something there like NET_CAP_ADMIN?